tests/components/plum_lightpad/test_init.py

```diff
@@ -0,0 +1,104 @@
+"""Tests for the Plum Lightpad config flow."""
+from aiohttp import ContentTypeError
+from requests.exceptions import HTTPError
+
+from homeassistant.components.plum_lightpad.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from tests.async_mock import Mock, patch
+from tests.common import MockConfigEntry
+
+
+async def test_async_setup_no_domain_config(hass: HomeAssistant):
+    """Test setup without configuration is noop."""
+    result = await async_setup_component(hass, DOMAIN, {})
+
+    assert result is True
+    assert DOMAIN not in hass.data
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5801702)